tn-cli/tn-cli.py

```diff
@@ -24,6 +24,7 @@
     import queue
 import random
 import re
+import readline
```

Comment:
Please add readline to requirements.txt.

Reply:
There is something called pyreadline which exists on windows but seems to be abandoned:
https://pypi.org/project/pyreadline/

Reply:
Good catch! Yes, I'll check pyreadline - we need to see if it's compatible with raw_input()/input().

Reply:
Please also look at https://github.com/prompt-toolkit/python-prompt-toolkit
It seems to be well-supported albeit huge.